src/app/messages/pl/messages.json

```diff
@@ -125,6 +125,7 @@
   "i18n:notification:toast:highlights:popup:load-failure": "Niektóre zakreślenia na tej stronie nie mogą zostać załadowane. Sprawdź połączenie z internetem i spróbuj ponownie.",
   "i18n:notification:toast:highlights:popup:print-failure": "Nie możemy wydrukować tej strony. Sprawdź połączenie z internetem i spróbuj ponownie.",
   "i18n:notification:toast:study-guides:load-failure": "Nie możemy załadować zawartości Kompendium na tej stronie. Sprawdź połączenie z internetem i spróbuj ponownie.",
+  "i18n:notification:toast:highlights:select-overlap": "Nakładające się podświetlenia nie są obsługiwane.",
```

Comment:
@paulinaszczucinska Can you check this translation of "Overlapping highlights are not supported" ?

Reply:
Sure @RoyEJohnson, it would be: "Nakładające się zaznaczenia nie są obsługiwane."